CHANGELOG.md

```diff
@@ -22,3 +22,4 @@ and this project adheres to
 - chore: bump madara image
 - dev: rename crate conformance-test-utils to hive-utils
 - dev: update code and integration tests to return correct datatype for get_logs
+- feat: add rpc docker image for hive testing
```

Comment:
let's do changelog newest changes at the top of the changelog file from now on

Reply:
Gotcha will reorder this 😎